android/app/lint-baseline.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<issues format="6" by="lint 8.5.2" type="baseline" client="gradle" dependencies="false" name="AGP (8.5.2)" variant="all" version="8.5.2">
+<issues format="6" by="lint 8.10.0" type="baseline" client="gradle" dependencies="false" name="AGP (8.10.0)" variant="all" version="8.10.0">
 
     <!--TODO(#855) see if this is no longer needed-->
     <issue
         id="InvalidPackage"
         message="Invalid package reference in library; not included in Android: `javax.xml.stream`. Referenced from `org.apache.tika.utils.XMLReaderUtils`.">
         <location
-            file="$GRADLE_USER_HOME/caches/modules-2/files-2.1/org.apache.tika/tika-core/3.1.0/6ba44a9ddf8f6f2b4bc88e8bc64269aea0424607/tika-core-3.1.0.jar"/>
+            file="$GRADLE_USER_HOME/caches/modules-2/files-2.1/org.apache.tika/tika-core/3.2.0/9232bb3c71f231e8228f570071c0e1ea29d40115/tika-core-3.2.0.jar"/>
     </issue>
 
 </issues>
```

Comment:
```
deps: Update file_picker to 10.2.0, from 10.1.9
```

I'm not following why the lint-baseline changes are necessary. At this commit, for me, `tools/check android` still fails with what looks like the original location string (`3.2.0/9232bb…`). Is there a command I should run?

Reply:
> […] is the new location string, though, right?

Er oops, I think I copy-pasted from the wrong thing (the lint report showed two things (1) a suppression that wasn't used because nothing matched, and (2) an error that wasn't caught by a suppression, and they looked very similar).

Anyway, it's passing now; not sure what changed 🤷‍♂️ :

```
Lint found no errors or warnings

1 errors/warnings were listed in the baseline file (/Users/chrisbobbe/dev/zulip-flutter/android/app/lint-baseline.xml) but not found in the project; perhaps they have been fixed?

Note: The baseline was created using a different target/variant than it was checked against.
Creation variant: lint
Current variant: lintVitalRelease
```

Reply:
(So, LGTM.)

Reply:
Sounds good; thanks for checking into it!